routes/postals.js

```diff
@@ -0,0 +1,22 @@
+const express = require("express");
+const router = express.Router();
+const connection = require("../config");
+
+// GET POSTAL
+
+router.get("/", (req, res) => {
+
+  const idPostal = req.params.id;
+
+  // connection à la base de données, et recuperation resultat
+  connection.query('SELECT * FROM postal WHERE id = ?', [idPostal], (err, results) => {
+    if (err || results.length === 0) {
+      console.log(err);
+      res.status(404).json({error: "Utilisateur introuvable", data: {}});
```

Comment:
pas besoin d'envoyer data si vous n'envoyer rien
```suggestion
      res.status(404).json({error: "Utilisateur introuvable"});
```